_capability/3-prototyping/index.md

```diff
@@ -24,27 +24,49 @@ These engineers build models to find out whether their designs will work. Aerosp
+ - To pad the prototype out, there are lots of [Lorem ipsum](https://en.wikipedia.org/wiki/Lorem_ipsum), which may look pretty, but it doesn't help the team or users understand the intent of the service being prototyped
+ - The prototype is entirely static with no dynamic behavior or navigation. This makes the prototype difficult to understand without the author guiding people through it, preventing it from being circulated to lots of users.
+ - Either by technical or license limitations, the prototype is only available to people situated with the development team.
 
-### Level 2
- - Designers are using the production development tools (such as HTML and CSS) to produce working prototypes
- - Prototypes are published to a 'live' environment, and deployed continuously throughout their development
- - The prototypes are used to supplement user research, with activities such as alpha-beta testing used to gather large amounts of supporting data  
+
+### Moving to level 1
+
+ - Paper prototypes are more useful then static WYSIWYG produced mockups. They take less time to produce, are easy to modify and can be distributed easily.
+ - Start to share your prototypes with a wider audience of users. This may mean that you need to rethink how you explain the flow of your prototypes, and what distribution mechanisms you use.
+ - Focus on the high-value parts of the prototype, and skip everything else. Large amounts of text or content that doesn't help to evaluate this approach to the MVP will probably distract people.
+ - Read up on [Sketching in Code](http://alistapart.com/article/sketchingincode), which gets you started on developing prototypes in the technologies used for the final product.
+
+# Level 1
+
+Prototypes are being used successfully during your service design process, and the team collaborate on the designs before they are given to developers to implement.
+
+### Indications that you're operating at level 1
+
+ - Designs are first tested on paper with the team and users before becoming digital. The paper designs go up on the wall for the team to refer to throughout the process.
+ - The designers and developers collaborate on the prototypes, ensuring that all aspects of the design are implementable and technically sound
+ - Developers implement the paper prototypes simple CSS, HTML and JavaScript, which introduces navigation and have just enough interactive components to demonstrate the particularly interesting functionality.
+ - Although they're throwaways, the prototypes are treated as important team technical assets and are stored in the team's version control repository.
+
+### Moving to level 2
+
+ - Start pair-programming with your designers when implementing coded prototypes. [There](http://www.w3schools.com/html/default.asp) are [heaps](http://www.w3schools.com/css/default.asp) of free learning resources online to help people learn (or re-learn) how to build content for the web.
```

Comment:
while w3schools seems to have finally update their site, I think there are better resources